packages/react/src/hooks/useSessionMessages.ts

```diff
@@ -124,13 +144,25 @@ export function useSessionMessages(session?: UseSessionReturn): UseSessionMessag
     }
   }, [receivedMessages]);
```

Comment:
issue (non-blocking): probably makes sense to use `sortedReceivedMessages` instead of `receivedMessages` ?